File: done/python/1131.py
# 1131 - Grenais
grenal = 1
scoreboard = [0, 0, 0]

while grenal != 2:
    inter, gremio = map(int, input().split())
    if inter > gremio:
        scoreboard[0] += 1
    elif gremio > inter:
        scoreboard[1] += 1
    elif gremio == inter:
        scoreboard[2] += 1
    grenal = int(input('Novo grenal (1-sim 2-nao)\n'))

print('{} grenais'.format(sum(scoreboard)))
print('Inter:{}'.format(scoreboard[0]))
print('Gremio:{}'.format(scoreboard[1]))
print('Empates:{}'.format(scoreboard[2]))

if scoreboard[0] > scoreboard[1]:
    print('Inter venceu mais')
elif scoreboard[0] < scoreboard[1]:
    print('Gremio venceu mais')
else:
    print('Não houve vencedor')

File: problems/c/1131.c
#include <stdio.h>

int main(void)
{
    int inter, gremio, novo_grenal, placar[3] = {0, 0, 0};

    do
    {
        scanf("%d %d", &inter, &gremio);

        if (inter > gremio)
        {
            placar[0]++;
        }
        else if (inter < gremio)
        {
            placar[1]++;
        }
        else
        {
            placar[2]++;
        }

        printf("Novo grenal (1-sim 2-nao)\n");
        scanf("%d", &novo_grenal);
    }
    while(novo_grenal == 1);

    printf("%d grenais\n", placar[0] + placar[1] + placar[2]);

    printf("Inter:%d\nGremio:%d\nEmpates:%d\n", placar[0], placar[1], placar[2]);

    if (placar[0] == placar[1])
    {
        printf("Nao houve vencedor\n");
    }
    else if (placar[0] > placar[1])
    {
        printf("Inter venceu mais\n");
    }
    else
    {
        printf("Gremio venceu mais\n");
    }

    return 0;
}
